privacy.adoc

```diff
@@ -189,7 +189,7 @@ ending there. (This isn't the only example of it, by far ... but its
 one of the more fun ones).
 ____
 
-In this case, all went well thanks to the philanthropically minded
+In this case, it all went well thanks to the philanthropically-minded
 hacker, but don't count on that next time.
 
 === Non-blockchain privacy
```

Comment:
If we go for technical xrefs we should write:

```
[[nonblockchainprivacy]]
=== Non-blockchain privacy
```

because we need that reference ID later on in the chapter. I'll add a comment at that place.